constellations/orion/src/repos/people_repo.go

```diff
@@ -0,0 +1,232 @@
+package repos
+
+import (
+	"database/sql"
+	"time"
+
+	"github.com/ahsu1230/mathnavigatorSite/constellations/orion/src/domains"
+)
+
+// Global variable
+var UserRepo UserRepoInterface = &userRepo{}
+
+// Implements interface userRepoInterface
+type userRepo struct {
+	db *sql.DB // golang native db connection
+}
+
+// Interface to implement
+type UserRepoInterface interface {
+	Initialize(db *sql.DB)
+	SelectAll(string, int, int) ([]domains.User, error)
+	SelectById(uint) (domains.User, error)
+	SelectByGuardianId(uint) ([]domains.User, error)
```

Comment:
Replace SelectByGuardianId with selectByFamilyId(), which will give us a list of users.